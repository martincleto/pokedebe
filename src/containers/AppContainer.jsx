import React, { Component } from 'react'

import App from '../core/App'

require('../stylesheets/app.scss')

class AppContainer extends Component {
  constructor() {
    super()
    this.state = {
      active: false,
    }

    this.data = null
  }

  getData(query) {  // eslint-disable-line no-unused-vars
    this.data = {
      someKey: 'someValue'
    }
  }

  render() {
    return (
      <App active={this.state.active} />
    )
  }
}

export default AppContainer
