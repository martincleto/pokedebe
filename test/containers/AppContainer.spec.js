import React from 'react'
import renderer from 'react-test-renderer'
import AppContainer from '../../src/containers/AppContainer'

describe('AppContainer', function() {
  test('should render without error', () => {
    const component = renderer.create(
      <AppContainer/>
    ).toJSON();

    expect(component).toMatchSnapshot();
  });
})
