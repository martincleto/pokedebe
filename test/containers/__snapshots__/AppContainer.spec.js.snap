// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContainer should render without error 1`] = `
<div
  className="container"
>
  <header>
    <h1
      className="heading heading--level-1"
    >
      Pok√©debe
    </h1>
  </header>
  <p>
    Search
  </p>
  <footer
    className="footer"
  >
    <p>
      Made with 
      <span>
        love
      </span>
       by 
      <a
        href="http://github.com/martincleto"
        title="martincleto at GitHub"
      >
        martincleto
      </a>
       to Enric
    </p>
  </footer>
</div>
`;
